src/Npgsql/SqlQueryParser.cs

```diff
@@ -47,87 +50,94 @@ internal void ParseRawQuery(string sql, bool standardConformantStrings, NpgsqlPa
             var parenthesisLevel = 0;
 
         None:
-            if (currCharOfs >= end) {
+            if (currCharOfs >= end)
+            {
```

Comment:
nit: omit here and similar

Reply:
Sorry not sure what you mean, you mean remove the brackets alltogether? Something like:

```cs
if (currCharOfs >= end)
    goto Finish;
```

Reply:
Yep, for consistency with other files. By "and similar" I meant that there are other instances to omit, just didn't want to tag all of them :)

Reply:
lol fair enough. I'll look the file over and change what I find.